exercises/custom-set/src/lib.rs

```diff
@@ -1 +1,61 @@
+use std::fmt::Debug;
+use std::marker::PhantomData;
 
+#[derive(Debug, PartialEq)]
+pub struct CustomSet<T> {
+    // This field is here to make the template compile and not to
+    // complain about unused type parameter 'T'. Once you start
+    // solving the exercise, delete this field and the 'std::marker::PhantomData'
+    // import.
+    phantom: PhantomData<T>,
+}
+
+impl<T: Debug> CustomSet<T> {
```

Comment:
OK to add `<T: Debug>` here, since a `T` is used in so many `unimplemented` messages? Or should it be left out?

Reply:
The problem with the `Debug` trait bound seems to appear in a lot of exercises. Instead of having separate discussion in every PR, would you mind creating an issue about it? So that there would be one fixed solution.